src/protocol/protocol.ts

```diff
@@ -786,6 +790,76 @@ export namespace Log {
   }
 }
 
+export namespace Network {
+  export type Event = BeforeRequestSentEvent;
+
+  export type BeforeRequestSentEvent = EventResponse<
+    EventNames.BeforeRequestSentEvent,
+    BeforeRequestSentParams
+  >;
+
+  type Header = {
+    name: string;
+    value?: string;
+    binaryValue?: number[];
+  };
+
+  type Cookie = {
+    name: string;
+    value?: string;
+    binaryValue?: number[];
+    domain: string;
+    path: string;
+    expires?: number;
+    size: number;
+    httpOnly: boolean;
+    secure: boolean;
+    sameSite: 'strict' | 'lax' | 'none';
+  };
+
+  type FetchTimingInfo = {
+    timeOrigin: number;
+    requestTime: number;
+    redirectStart: number;
+    redirectEnd: number;
+    fetchStart: number;
+    dnsStart: number;
+    dnsEnd: number;
+    connectStart: number;
+    connectEnd: number;
+    tlsStart: number;
+    requestStart: number;
+    responseStart: number;
+    responseEnd: number;
+  };
+
+  type RequestData = {
+    request: string;
+    url: string;
+    method: string;
+    headers: Header[];
+    cookies: Cookie[];
+    headersSize: number;
+    bodySize: number | null;
+    timings: FetchTimingInfo;
+  };
+
+  type BaseEventParams = {
+    context: string | null;
+    isRedirect: boolean;
+    navigation: BrowsingContext.Navigation | null;
+    redirectCount: number;
+    request: RequestData;
+    timestamp: number;
+  };
+
+  export type BeforeRequestSentParams = BaseEventParams & {};
+
+  export enum EventNames {
```

Comment:
Missing: `AllEvents` definition.

Also ensure to update [`unrollEvents`](https://github.com/GoogleChromeLabs/chromium-bidi/blob/f777c628097e0a17664674a587f302c952c0a3cf/src/bidiMapper/domains/events/SubscriptionManager.ts#L42)

Reply:
`AllEvents` is in line 971

Reply:
done